mozregression/history.py

```diff
@@ -0,0 +1,46 @@
+"""
+Representation of the bisection history.
+"""
+
+
+class BisectionStep(object):
+    """
+    An history bisection step.
+    """
+    def __init__(self, build_range, index, verdict=None):
+        self.build_range = build_range
+        self.index = index
+        self.verdict = verdict
+
+
+class BisectionHistory(object):
```

Comment:
You might consider making this class inherit from list, and just adding one or two custom methods for convenience. Something like:

```
class BisectionHistory(list):
    def add_step(self, build_range, index, verdict):
        self.append(BisectionStep(...))
```